app/presenters/bill-runs/review/review-licence.presenter.js

```diff
@@ -180,7 +181,7 @@ function _formatReviewReturns(reviewReturns) {
       abstractionPeriod: formatAbstractionPeriod(periodStartDay, periodStartMonth, periodEndDay, periodEndMonth),
       description,
       issues: formatIssues(issues),
-      purpose: purposes[0].tertiary.description,
+      purpose: formatPurposes(purposes),
```

Comment:
If you do array.join() you wouldn't need that in the base presenter so it could just be put into there

Reply:
It wouldn't work on this as `purposes` is an array of objects, and the string I want is buried in the object